<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QTableWidget" name="tableWidget">
                <property name="geometry">
                    <rect>
                        <x>150</x>
                        <y>330</y>
                        <width>481</width>
                        <height>131</height>
                    </rect>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton">
                <property name="geometry">
                    <rect>
                        <x>510</x>
                        <y>100</y>
                        <width>81</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Angsana New</family>
                        <pointsize>14</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>บันทึก</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit">
                <property name="geometry">
                    <rect>
                        <x>160</x>
                        <y>100</y>
                        <width>321</width>
                        <height>41</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="label">
                <property name="geometry">
                    <rect>
                        <x>160</x>
                        <y>140</y>
                        <width>241</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Angsana New</family>
                        <pointsize>18</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>รหัสนักศึกษา/rfid</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_2">
                <property name="geometry">
                    <rect>
                        <x>160</x>
                        <y>220</y>
                        <width>101</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Angsana New</family>
                        <pointsize>16</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>รหัสนักศึกษา</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_3">
                <property name="geometry">
                    <rect>
                        <x>160</x>
                        <y>270</y>
                        <width>101</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Angsana New</family>
                        <pointsize>16</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>ชื่อ</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_4">
                <property name="geometry">
                    <rect>
                        <x>360</x>
                        <y>270</y>
                        <width>101</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Angsana New</family>
                        <pointsize>16</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>นามสกุล</string>
                </property>
            </widget>
            <widget class="Line" name="line">
                <property name="geometry">
                    <rect>
                        <x>150</x>
                        <y>310</y>
                        <width>481</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
</widget>
    <resources/>
    <connections/>
</ui>